### ISSUE TITLE
5s timeout for beaconHTTPClient is not enough

### ISSUE THREAD

**Issue by MrKoberman** [flashbots/mev-boost-relay#411]
We are experiencing a lot of context timeouts when fetching `active, pending` validators from beacon. This could potentially be an ENV variable or have a separate http client just to get validators. 

Did anyone else experience this timeout as well?

**Comment by austonst** [comment]
Yes, also experiencing this. Looks like we opened issues for it at the same time, ours is #412 .
